PHPFIleHandlingAndSessions/database.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+	$db=new PDO('mysql:host=localhost;port=3306;dbname=BooksDetails','root','root');
```

Comment:
I've asked you all to keep the DB name, DB columns and table names underscore_separated.